### PR TITLE
Works and about me

### PR DESCRIPTION
**Pull request by AckonSamuel**
Linters helped in detecting the programmable errors.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -26,24 +26,181 @@
...
+
+
+<div class="Hallelujah">
+    
+    <section class="Works">
+        <div class="Project-name"><h2>Projects</h2></div>
+        <div class="work one">
+            <div class="project photo"><img src="assets/work/work_one.png" class="work_one" alt="#"></div>
+            <div class="project summary">
+                <div class="nameP"><p>Project name goes here</p></div>
```
Comment (elmar8287) [inline index.html:53]: - [ ] Please, consider that the container of the section is a `section` tag. In your case, you insert all code inside the `div` tag (start from line 46). Please, do not use the `div` tag for the whole section. Use the `section` tag for works.

Comment (elmar8287) [inline index.html:53]: - [ ] Please, use for each project title (in your case is "Project name goes here") use a header tag `hx` (e.g.h2, h3...)

File: index.html [hunk 2]
```diff
@@ -26,24 +26,181 @@
...
+                <div class="technology">
+                    <div class="tech1"><p>HTML/CSS</p></div>
+                    <div class="tech1"><p>Ruby on Rails</p></div>
+                    <div class="tech1"><p>JavaScript</p></div>
+                </div>
+                <div class="project-view">
+                    <div class="word"><p>See this project</p></div>
+                    <div class="union"><img src="assets/work/Union.png" class="arrow" alt="#"></div>
```
Comment (elmar8287) [inline index.html:61]: - [ ] Please, consider that the button (e.g., "See this project") is a link `a` or `button` tag

File: index.html [hunk 3]
```diff
@@ -26,24 +26,181 @@
...
+        <div class="Project-name"><h2>Projects</h2></div>
+        <div class="work one">
+            <div class="project photo"><img src="assets/work/work_one.png" class="work_one" alt="#"></div>
+            <div class="project summary">
+                <div class="nameP"><p>Project name goes here</p></div>
+                <div class="technology">
+                    <div class="tech1"><p>HTML/CSS</p></div>
+                    <div class="tech1"><p>Ruby on Rails</p></div>
+                    <div class="tech1"><p>JavaScript</p></div>
+                </div>
```
Comment (elmar8287) [inline index.html:58]: - [ ] Please, consider that the list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list `ul` and each of them a `li`

File: index.html [hunk 4]
```diff
@@ -26,24 +26,181 @@
...
+            adipiscing elit, sed do eiusmod 
+            tempor incididunt ut labore et 
+            dolore magna aliqua. Ut enim ad 
+            minim veniam, quis nostrud 
+            exercitation ullamco
+             laboris nisi ut aliquip ea eiusmod</p>
+            </div>
+            <div class="getflex">
+        <div class="cvBox"><p>Get my CV</p>
+        <div class="union_1"><img src="assets/Union_two.png" class="arrow_down" alt="#"></div></div>
```
Comment (elmar8287) [inline index.html:167]: - [ ] Please, consider that the button (e.g., "Get my CV") is a link `a` or `button` tag

File: index.html [hunk 5]
```diff
@@ -26,24 +26,181 @@
...
+                </ul>
+            </div>
+            <div class="Frameworks">
+                <h4>Frameworks</h4>
+                <ul>
+                        <li><p>Bootstrap</p></li>
+                        <li><p>Ruby on rails</p></li>
+                        <li><p>RSpec</p></li>
+                        <li><p>Capibara</p></li>
+                    </ul>
```
Comment (elmar8287) [inline index.html:185]: - [ ] Please, consider that the technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li`

File: index.html [hunk 6]
```diff
@@ -26,24 +26,181 @@
...
+        <div class="union_1"><img src="assets/Union_two.png" class="arrow_down" alt="#"></div></div>
+    </div>
+        <div class="gridContainer">
+            <div class="lang"><h4>Languages</h4>
+                <ul>
+                    <li><p>JavaScript</p></li>
+                    <li><p>Ruby</p></li>
+                    <li><p>HTML</p></li>
+                    <li><p>CSS</p></li>
+                </ul>
```
Comment (elmar8287) [inline index.html:176]: This is correct for that the technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list `ul` and each of them a `li` - Nice job 👍

File: index.html [hunk 7]
```diff
@@ -26,24 +26,181 @@
...
         Taka a look of my works. 
         If you like what you see and have a 
         project you need coded,
-         don’t hesitate and contact me</h4></div>
+         don’t hesitate and contact me</p></div>
        <div class="rectangle">
           <div>Start collaboration</div></div>
 
        <div class="designs">
           <div class="image3"><img src="assets/image_geometry_2.png" alt="#" class="shape002"></div>
```
Comment (elmar8287) [inline index.html:40]: - [ ] Please, consider that the section has the proper background image (decorations are always backgrounded images, not `img` tags). So you can do it in `.css` file using `background-image: url("your img path");`

**Review by elmar8287 — CHANGES_REQUESTED** [review]
Hi @AckonSamuel ,

Great work on making the project requirements 👏 

There are still some requirements that aren't done yet, please work on it and ask for another review.
      
 ## Required Changes ♻️

- [ ] Please, consider comments under review
- [ ] Please, modify your README file according to MICROVERSE template.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: index.html (hunk 1 above)
Comment (Kingstalux) [reply index.html]: This issue has not yet been resolved.

File: index.html (hunk 4 above)
Comment (Kingstalux) [reply index.html:167]: This issue has not been resolved.

File: index.html (hunk 7 above)
Comment (Kingstalux) [reply index.html]: This issue has not been resolved.

File: index.html (hunk 6 above)
Comment (AckonSamuel) [reply index.html:183]: Please, I just made the changes

**Review by Kingstalux — CHANGES_REQUESTED** [review]
Hi @AckonSamuel , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
- The following issues from the previous review have not been resolved:
- [x] The title for each of your projects (in your case it is 'Project name goes here') is supposed to be inside an `h2` or `h3` tag but it is currently inside a `p` tag. It would be good to correct that for best semantic practices.
d3
- [x] 'Get my CV' is a button or a link that takes the user to your CV. It would be great if you could change that.
d4
- [x] Your technologies section is lacking the appropriate background images. It would be nice if you could add that. Also, the padding of your elements does not follow the design. Do well to rectify that too.
- Your design:
d2
- Figma design:
d1
- [ ] Your README file is basically empty at this point. To make your repo more professional, copy this [template](https://github.com/microverseinc/readme-template) provided by Microverse and edit it with your information and information about your project.
- [x] It would be nice to write meaningful commit messages which say the different changes you've implemented in a commit. It makes your project easier to follow up in the future. Do well to follow this.
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 6 above)
Comment (Kingstalux) [reply index.html:183]: One last thing, you are not supposed to make changes to your repo after you've submitted it for review. Wait for the review comments before making any further changes after submitting your project.

**Review by Kingstalux — APPROVED** [review]
Hi @AckonSamuel , 
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

Good job applying the requested changes 👍  Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏
- [ ] (OPTIONAL) Your README file is a little better now but it can still be improved upon. It would be nice to pay more attention to that in your upcoming projects.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**